tests/regression/lp-1898038/test-multipass-support-app/bin/test-multipass-support

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
I think you can just do `/bin/sh` here, the script will work either way.

Reply:
Thanks, fixed in 8103f2964c